docs/writing_a_parser_plugin/references/old_docs.md

```diff
@@ -0,0 +1,338 @@
+# How to write a parser
```

Comment:
Is this copied from pre-existing documentation? I will be skipping it then.

Reply:
Yes, this is just for our reference as we write the new docs, I will remove later.